scripts/zkai_attention_kv_d8_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,682 @@
+        "source_plus_sidecar_raw_proof_bytes",
+        "fused_proof_size_bytes",
+        "fused_over_source_proof_bytes",
+        "fused_saves_vs_source_plus_sidecar_bytes",
+        "fused_to_source_plus_sidecar_ratio",
+        "mutation_results",
+        "mutations_checked",
+        "mutations_rejected",
+    }
+    missing = required - set(result)
+    if missing:
+        raise AttentionKvD8FusedSoftmaxTableGateError(f"missing result keys: {sorted(missing)}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)